plugins/modules/hpilo_boot.py

```diff
@@ -76,6 +76,11 @@
     default: TLSv1
     type: str
     choices: ["SSLv3", "SSLv23", "TLSv1", "TLSv1_1", "TLSv1_2"]
+  idempotent_boot_once:
+    description:
+      - "This makes the boot_once idempotent see github PR: https://github.com/ansible-collections/community.general/pull/9646"
```

Comment:
Sorry, one more nit to be picked.
```suggestion
      - "This makes the boot_once idempotent see github PR: U(https://github.com/ansible-collections/community.general/pull/9646)."
```